firmware/hw_layer/stepper.cpp

```diff
@@ -188,8 +188,12 @@ bool StepDirectionStepper::pulse() {
 	return true;
 }
 
-void StepperHw::pause() const {
-	chThdSleepMicroseconds((int)(MS2US(m_reactionTime)));
+void StepperHw::sleep(void) {
+	pause();
+}
+
+void StepperHw::pause(int divisor) const {
+	chThdSleepMicroseconds((int)(MS2US(m_reactionTime)) / divisor);
```

Comment:
We have to be a little careful here - since we use tick mode at 1khz, we can't sleep less than 1ms.  If you try to sleep less than 1ms, it will sleep for 0, essentially just a `yield()`, potentially starving lower priority threads.

Reply:
see #3214

Reply:
Maybe `sleepMs(max(1, reactionTime/divisor))` to ensure we always sleep